core/src/main/java/org/kohsuke/stapler/framework/io/LargeText.java

```diff
@@ -509,8 +515,8 @@ public boolean isFull() {
      * Represents the read session of the {@link Source}.
      * Methods follow the contracts of {@link InputStream}.
      */
-    private interface Session extends Closeable {
-        long skip(long start) throws IOException;
+    public interface Session extends Closeable {
+        long skip(long n) throws IOException;
```

Comment:
can we document these methods now that the interface is public?

Reply:
Done in 50ce95ca28a81a072262f166a08cf85799045f83